src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -36,6 +36,11 @@
     short-summary: Manage build definitions, which can be triggered by git commits or base image updates.
     """
 
+helps['acr image'] = """
```

Comment:
Will there be additional commands within the `acr image` group?  If not, this should be `acr import-image` or potentially `acr import`.  If there will be additional commands in this group (even if its a future PR) then an `image` group is appropriate.

Reply:
How is this different from the `repository` group which says "Manage repositories (image names) for Azure Container Registries"?

Reply:
ping @NatalieGong